File: NthTerm.c
#include <stdio.h>
#include <math.h>

int main() {
    int a, r, n;
    double term;

    printf("Enter first term (a): ");
    scanf("%d", &a);

    printf("Enter common ratio (r): ");
    scanf("%d", &r);

    printf("Enter term number (n): ");
    scanf("%d", &n);

    term = a * pow(r, n - 1);

    printf("The %dth term of the GP is: %.0f", n, term);

    return 0;
}

File: NthTerm.java
import java.util.Scanner;

public class GPNthTerm {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter first term (a): ");
        int a = sc.nextInt();

        System.out.print("Enter common ratio (r): ");
        int r = sc.nextInt();

        System.out.print("Enter term number (n): ");
        int n = sc.nextInt();

        double term = a * Math.pow(r, n - 1);

        System.out.println("The " + n + "th term of the GP is: " + term);

        sc.close();
    }
}
